src/main/java/org/jenkinsci/plugins/gitserver/FileBackedHttpGitRepository.java

```diff
@@ -133,7 +133,7 @@ public void onPostReceive(ReceivePack rp, Collection<ReceiveCommand> commands) {
 
     /**
      * Called when new ref is pushed to update the {@linkplain #workspace local workspace}.
-     * The default implementation does "git reset --hard master"
+     * The default implementation does "git reset --hard controller"
```

Comment:
this one is git related (and not Jenkins related) so as far as I know it can be changed by "main" as main branch.
```suggestion
     * The default implementation does "git reset --hard main"
```

Reply:
I agree with @aHenryJard .  This is a reference to command line git. `main` is the better choice.